.github/workflows/release.yml

```diff
@@ -37,16 +79,18 @@ jobs:
       #     with only pre-release tag, e.g. `x.y.z-beta.67`, and not the others
       # Flavor:
       #    `latest` will tag the image with `latest`
+
       - name: Extract GIT metadata for Docker
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: registry.cern.ch/${{ github.repository_owner }}/almalinux
+          images: |
+            ${{ env.REGISTRY }}/${{ github.repository_owner }}/almalinux
           tags: |
-            type=semver,pattern={{major}}
-            type=semver,pattern={{major}}.{{minor}}
+            # type=semver,pattern={{major}}
```

Comment:
Why commenting this out?

Reply:
I am not sure whether we want/need a tag for major version.